pkg/transforms/policy.go

```diff
@@ -130,7 +130,108 @@ func OperatorPolicyResourceBuilder(c *unstructured.Unstructured) *PolicyResource
 	node.Properties["deploymentAvailable"] = deploymentAvailable
 	node.Properties["upgradeAvailable"] = upgradeAvailable
 
-	return &PolicyResource{node: node}
+	return &StandalonePolicyResource{
+		node:   node,
+		policy: *c,
+	}
+}
+
+func ConfigPolicyResourceBuilder(c *unstructured.Unstructured) *StandalonePolicyResource {
+	node := transformCommon(c) // Start off with the common properties
+	node = recordRelatedObjects(c, node)
+
+	return &StandalonePolicyResource{
+		node:   getPolicyCommonProperties(c, node),
+		policy: *c,
+	}
+}
+
+func recordRelatedObjects(c *unstructured.Unstructured, node Node) Node {
+	relatedObjects, found, err := unstructured.NestedSlice(c.Object, "status", "relatedObjects")
+	if !found || err != nil {
+		return node
+	}
+
+	// The node metadata must be a map[string]string, so this puts all related
+	// objects in one entry. This info is *only* to update the metadata in order
+	// to trigger the edges to be re-calculated.
```

Comment:
Is this data only to **trigger** the edge logic? Or is the data here actually used to re-calculate edges?

Reply:
Only to trigger the edge logic. The same data is pulled out separately in `BuildEdges`.

It would be possible for `BuildEdges` to parse the metadata string (or update metadata to accept non-strings), if that would be better for some other reason

Reply:
Thanks for the updates!